hypothesis_bio/hypothesis_bio.py

```diff
@@ -409,12 +409,18 @@ def nanopore_sequence_identifier(draw) -> str:
 
 @composite
 def fastq_quality(
-    draw, size=0, min_score: int = 0, max_score: int = 93, offset: int = 33
+    draw,
+    min_length: int = 1,
+    max_length: int = None,
```

Comment:
I'm pretty sure Mypy is going to complain in the future about this. Try `Optional[int]`